### PR TITLE
Cleaning up the doc after moving some parts of doc to libbeat

### PR DESCRIPTION
**Pull request by monicasarbu**

### CODE REVIEW

**Comment by monicasarbu** [comment]
Required by https://github.com/elastic/libbeat/issues/85

File: docs/configuration.asciidoc [hunk 1]
```diff
@@ -1,133 +1,15 @@
 [[configuration]]
 == Configuration
 
-The Packetbeat configuration file uses
-http://yaml.org/[YAML] for its syntax. It consists of the following sections:
+The Packetbeat configuration file useshttp://yaml.org/[YAML] as any other Beat.
```
Comment (tsg) [inline docs/configuration.asciidoc]: missing space between uses and http?

File: docs/configuration.asciidoc [hunk 2]
```diff
@@ -1,133 +1,15 @@
 [[configuration]]
 == Configuration
 
-The Packetbeat configuration file uses
-http://yaml.org/[YAML] for its syntax. It consists of the following sections:
+The Packetbeat configuration file useshttp://yaml.org/[YAML] as any other Beat. 
+The common configuration options to all Beats are described TODO[here].
```
Comment (tsg) [inline docs/configuration.asciidoc]: just please remember to remove the TODOs.

File: docs/configuration.asciidoc [hunk 3]
```diff
@@ -680,253 +547,7 @@ can be used for sharing the topology. If you have both Redis and Elasticsearch
...
-
-Maximum size in kilobytes of each file. When this size is reached, the files are
-rotated. The default value is 10 MB.
-
-===== number_of_files
-
-Maximum number of files under path. When this number of files is reached, the
-oldest file is deleted and the rest are shifted from last to first. The default
-is 7 files.
+More details about the Beats outputs are available TODO[here].
```
Comment (tsg) [inline docs/configuration.asciidoc]: Shouldn't we at least have an Outputs header instead of just a single line that most people will miss?

**Pull request merged**